pkg/products/ups/reconciler.go

```diff
@@ -107,34 +104,34 @@ func (r *Reconciler) VerifyVersion(installation *integreatlyv1alpha1.RHMI) bool
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1alpha1.RHMI, product *integreatlyv1alpha1.RHMIProductStatus, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-	logrus.Infof("Reconciling %s", defaultUpsName)
+	r.log.Info("Start Reconcile")
```

Comment:
Feels like this could be a bit more descriptive. "Starting UPS Reconcile" ?

Reply:
Log will output with context product=UPS